README.md

```diff
@@ -159,14 +159,22 @@ pytest
                   Erik Bjäreholt and
                   Maria Sayu Yamamoto and
                   Pierre Clisson and
-                  Marie-Constance Corsi},
-  title        = {pyRiemann/pyRiemann: v0.5},
-  month        = june,
-  year         = 2023,
+                  Marie-Constance Corsi and
+                  Igor Carrara and
+                  Apolline Mellot and
+                  Bruna Junqueira Lopes and
+                  Brent Gaisford and
+                  Ammar Mian and
+                  Anton Andreev and
+                  Gregoire Cattan and
+                  Arthur Lebeurrier},
+  title        = {pyRiemann},
+  month        = oct,
+  year         = 2024,
+  version      = {v0.7},
   publisher    = {Zenodo},
-  version      = {v0.5},
-  doi          = {10.5281/zenodo.8059038},
-  url          = {https://doi.org/10.5281/zenodo.8059038}
+  doi          = {10.5281/zenodo.593816},
+  url          = {https://doi.org/10.5281/zenodo.593816}
```

Comment:
With this url, the DOI always points towards latest version.
So we only need to add new contributors when preparing a release, and update version number+date.

Reply:
Thx @sylvchev !

Reply:
done